charts/cray-certmanager/templates/cert-manager/securityContext-job.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ template "cray-certmanager.fullname" . }}-cert-manager-{{ .Release.Revision }}"
+  labels:
+    app.kubernetes.io/name: {{ template "cray-certmanager.fullname" . }}-cert-manager
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
+    spec:
+      restartPolicy: Never
+      serviceAccountName: "cert-manager"
+      containers:
+        - name: "cert-manager"
+          image: "artifactory.algol60.net/csm-docker/stable/docker-kubectl:1.19.15"
+          command:
+            - '/bin/sh'
+          args:
+            - '-c'
+            - 'until kubectl get -n cert-manager deployments cray-certmanager-cert-manager > /dev/null 2>&1; do echo "Waiting for cray-certmanager-cert-manager deployments to be created"; sleep 5; done; /usr/local/sbin/modify_securityContext.sh'
```

Comment:
If this was a post-install/post-upgrade hook then this can be simplified because the cray-certmanager-cert-manager deployment will already exist and there will be no need to wait for it to be created.